src/app/core/services/firebaseAnalytics.service.ts

```diff
@@ -2,12 +2,19 @@ import { Injectable } from '@angular/core'
 import { FirebaseAnalytics } from '@ionic-native/firebase-analytics/ngx'
 
 import { User } from '../../shared/models/user'
+import { Utility } from '../../shared/utilities/util'
 
 @Injectable()
 export class FirebaseAnalyticsService {
-  constructor(private firebaseAnalytics: FirebaseAnalytics) {}
+  constructor(
+    private firebaseAnalytics: FirebaseAnalytics,
+    private utility: Utility
+  ) {}
 
   logEvent(event: string, params): Promise<any> {
+    console.log('Event', event)
+    if (this.utility.isPlatformBrowser)
```

Comment:
@nivemaham I have added a platform check if the app is running on the browser in the FirebaseAnalytics service so we don't have to add additional functions.